src/Parallel/Actions/SetupDataBox.hpp

```diff
@@ -41,12 +41,22 @@ struct optional_compute_tags<Action,
 };
 
 template <typename ActionList>
-using get_action_list_simple_tags = tmpl::remove_duplicates<
-    tmpl::flatten<tmpl::transform<ActionList, optional_simple_tags<tmpl::_1>>>>;
+using get_action_list_simple_tags =
```

Comment:
replace the `get_action_list_simple_tags` (and compute) with structs that define these as a `type` type alias so you don't need to use `tmpl::bind` below